src/Message/ReportingResponse.php

```diff
@@ -84,4 +95,87 @@ public function getSubscriptions()
         }
         return $subscriptions;
     }
+
+    /**
+     * @return Refund[]|null
+     * @throws Exception
+     */
+    public function getRefunds()
+    {
+        if (!empty($this->refunds)) {
+            return $this->refunds;
+        }
+
+        if (!is_array($this->data) || !isset($this->data['data'])) {
+            return null;
+        }
+
+         /** @var array<string, string> $row */
+        foreach ($this->data['data'] as $row) {
+            /**
+             * If ReportingFetchTransactionsRequest::setTransactionType(Constants::TRANSACTION_TYPE_REFUND) was used
+             * then these should always be of type 'Refund'. This check is just added as extra layer of safety to
+             * ensure only refunded transactions are included because excluding a transaction type will return back
+             * all types.
```

Comment:
`"excluding a transaction type will return back all types"` - is this referring to excluding a transaction type on the request?

Reply:
That's correct. This means that if no type is used then all types will be returned by default (sales, chargebacks, refunds)